systemservices/marketplace/mesg.yml

```diff
@@ -0,0 +1,414 @@
+                  purchaser: &purchaser
+                    name: ""
+                    description: ""
+                    type: String
+                  expire: &expire
+                    name: ""
+                    description: ""
+                    type: String
+                  createTime: *createTime
+      error: &outputError
+        name: "Error"
+        description: "Output when an error occurs"
+        data:
+          message:
+            name: "Message"
+            description: "The error message"
+            type: String
+  getService:
+    name: ""
+    description: ""
+    inputs:
+      sid: &inputSid
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data: *service
+      error: *outputError
+  getServiceVersion:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      hash: &inputHash
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data: *version
+      error: *outputError
+  serviceExist:
+    name: ""
+    description: ""
+    inputs:
+      sid: &inputSid
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data:
+          exist:
+            name: ""
+            description: ""
+            type: Boolean
+      error: *outputError
+  createService:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      from: &inputFrom
+        name: ""
+        description: ""
+        type: String
+      gas: &inputGas
+        name: ""
+        description: ""
+        type: Number
+        optional: true
+      gasPrice: &inputGasPrice
+        name: ""
+        description: ""
+        type: String
+        optional: true
+    outputs:
+      error: *outputError
+      success: &outputTransaction
+        data:
+          chainID:
+            name: "Chain ID"
+            description: "The ID of the chain the transaction is for. See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids."
+            type: Number
+          nonce:
+            name: "Nonce"
+            description: "Nonce to use."
+            type: Number
+          to:
+            name: "To"
+            description: "The to address."
+            type: String
+          value:
+            name: "Value"
+            description: "The value in Wei."
+            type: String
+          gas:
+            name: "Gas"
+            description: "The maximum gas to use to execute this transaction"
+            type: Number
+          gasPrice:
+            name: "Gas price"
+            description: "The gas price in Wei."
+            type: String
+          data:
+            name: "Data"
+            description: "The data of the transaction."
+            type: String
+  createServiceOffer:
```

Comment:
We could keep this simple and not manage the offers

Reply:
We have the manage offer. It's a basic functionality.

Reply:
this can also be done in the marketplace website. They put their services to the marketplace and then we give them a link on the marketplace and they have to add the offer on top of their service with a nice ui